internal/desktop/systray_nonlinux.go

```diff
@@ -1,5 +1,5 @@
-//go:build windows || darwin || !linux
-// +build windows darwin !linux
+//go:build windows || darwin
+// +build windows darwin
```

Comment:
I am not sure if it is related to the build error, why didnt you just add `!freebsd` instead of removing the !linux part ?
eg
```go
//go:build windows || darwin || !linux || !freebsd
// +build windows darwin !linux !freebsd
```

Reply:
This does not compile at my system. So I decided to make an explicit selection. This makes only a difference, if there are more than these 4 systems (windows darwin linux freebsd).

Reply:
I think it somehow interferes with the windows cross build, not sure though